README.md

```diff
@@ -10,7 +10,7 @@ If you like what you see and have a project you need coded, don't hesitate to co
 
```

Comment:
- [ ] The _script option_ of your _package.json_ file is **not properly configured** and hence your application does not launch in a browser when the command `npm start` is run. Kindly append this line below to the _script options:_ 
`"start": "webpack serve --open"`, in your _packge.json_ file.